.github/workflows/test-and-deploy.yml

```diff
@@ -0,0 +1,73 @@
+name: Test and Deploy
```

Comment:
Also need to update the badge in the readme

Reply:
Done